examples/arc.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+#
+# Test for a push-button monome arc
+
+import asyncio
+import monome
+
+class ExampleArc(monome.Arc):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+class ExampleArcApp(monome.ArcApp):   
+    def __init__(self):
+        arc = ExampleArc()
+        self.pos = [0, 0, 0, 0];
+        super().__init__(arc)
```

Comment:
consider moving the superclass constructor call to the top of the subclass constructor, as subsequent statements might use device-specific internal member variables, which are initialized in the superclass.